test-dgidb.R

```diff
@@ -0,0 +1,5 @@
+source("dgidb.R")
+raw_gene_data <- get_interactions(c("BRAF","PDGFRA"),use_processing=FALSE)
+gene_data <- get_interactions(c("BRAF","PDGFRA"))
+raw_drug_data <- get_interactions(c("SUNITINIB","ZALCITABINE"),use_processing=FALSE,search='drugs')
+drug_data <- get_interactions(c("SUNITINIB","ZALCITABINE"),search='drugs')
```

Comment:
is there any kind of test suite we can implement for running/managing tests? I am unfamiliar with how tests work in R, is there something that you just essentially just run all tests and it will tell you which pass and fail?